DSC-JSS-NOIDA/HackHive
# fix(navbar): replace links with buttons and add onClick navigation

Closes:#154

**Description:**

This PR addresses the issue where the navigation bar links on the HackHive website were not functioning correctly. Specifically, clicking on the navigation links did not redirect to the intended sections or pages.

**Changes Made:**

- Replaced anchor tags with button elements in the Navbar.jsx file.
- Added onClick handlers to buttons to navigate using the useNavigate hook from react-router-dom.

**Steps to Test:**

- Pull the branch and run the project locally.
- Verify that clicking on the navigation links redirects to the appropriate pages.

**Expected Behavior:**

Navigation links should redirect to their respective sections or pages without issues.

**Deployment Issue:**

Even though the links redirect properly locally, they do not work when deployed on Netlify. This might be a deployment issue. The login button and the nav links follow the same method to navigate through the DOM, but the login button works while the nav links do not.I verified that the routes are defined properly or not,and found that they are correct,so I am not completely able to understand the reason behind the unexpected behaviour at the moment.

**Additional Information:**

N/A
